lib/routes.js

```diff
@@ -35,8 +35,8 @@ const statsClient = new StatsClient(localCacheClient, STATS_INTERVAL,
     STATS_EXPIRY);
 
 function checkUnsuportedRoutes(req, res, log) {
-    if (req.query.policy !== undefined ||
-        req.query.tagging !== undefined) {
+    if (utils.isUnsupportedQuery(req.queryStr)) {
```

Comment:
I looked at [routePOST](https://github.com/scality/S3/blob/master/lib/routes/routePOST.js) and saw that it effectively does this check already (if none of the three supported queries are defined, it returns NotImplemented). But do you think `routes.js` is a better place to filter out those requests?